src/agoradatatools/etl/transform/gene_info.py

```diff
@@ -70,7 +71,7 @@ def transform_gene_info(
         df=druggability, grouping="geneid", new_column="druggability"
     )
     druggability.rename(columns={"geneid": "ensembl_gene_id"}, inplace=True)
-    
+
```

Comment:
Should we add tests for this update?

Reply:
@thomasyu888 we haven't added any transform_gene_info tests yet, but when we do we will ensure that we cover these new fields.